shuup/core/pricing/_priceful.py

```diff
@@ -172,21 +182,15 @@ def taxful_price(self):
         :rtype: TaxfulPrice
         """
         price = self.price
-        if price.includes_tax:
-            return price
-        else:
-            return TaxfulPrice(price.amount + self.tax_amount)
+        return bankers_round((price if price.includes_tax else TaxfulPrice(price.amount + self.tax_amount)), 2)
 
     @property
     def taxless_price(self):
         """
         :rtype: TaxlessPrice
         """
         price = self.price
-        if price.includes_tax:
-            return TaxlessPrice(price.amount - self.tax_amount)
-        else:
-            return price
+        return bankers_round((TaxlessPrice(price.amount - self.tax_amount) if price.includes_tax else price), 2)
```

Comment:
What's the use for `price` after this? if anyone ever uses that, (s)he will end up having wrong values. Should `price` be private?